test/e2e/testnet/node.go

```diff
@@ -296,10 +300,23 @@ func (n Node) Client() (*http.HTTP, error) {
 }
 
 func (n *Node) Start() error {
-	if err := n.Instance.Start(); err != nil {
+	if err := n.StartAsync(); err != nil {
+		return err
+	}
+	if err := n.WaitUntilStartedAndForwardPorts(); err != nil {
 		return err
 	}
+	return nil
+}
+
+func (n *Node) StartAsync() error {
+	if err := n.Instance.StartAsync(); err != nil {
+		return err
+	}
+	return nil
+}
 
+func (n *Node) WaitUntilStartedAndForwardPorts() error {
```

Comment:
This can be done in a follow up but this function should really be two different functions since it performs two different actions.

In fact, I would be more in favour of grouping `StartAsync` and `WaitInstanceIsRunning` together

Reply:
Tracking issue https://github.com/celestiaorg/celestia-app/issues/3599